src/woodbury_pd_mat.jl

```diff
@@ -84,3 +84,7 @@ end
 function PDMats.unwhiten!(r::DenseVecOrMat, W::WoodburyPDMat{<:Real}, x::DenseVecOrMat)
     return unwhiten!(r, PDMat(Symmetric(Matrix(W))), x)
 end
+
+# NOTE: the parameterisation to scale up the Woodbury matrix is not unique. Here we
+# implement one way to scale it.
+*(a::WoodburyPDMat, c::T) where T<:Real = WoodburyPDMat(a.A, a.D * c, a.S * c)
```

Comment:
```suggestion
*(a::WoodburyPDMat, c::Real) = WoodburyPDMat(a.A, a.D * c, a.S * c)
```
I think this would probably be fine. Or were you having type inference problems?